smart-contracts/Cargo.toml

```diff
@@ -32,6 +33,8 @@ cw20-base = {version = "1.1.2", features = ["library"]}
 cw20-staking = "0.11.1"
 apollo-cw-asset = "0.1.2"
 cw-dex = {git = "https://github.com/quasar-finance/cw-dex", branch = "feat/deprecate-osmo-gamm"}
+mars-owner = "2.0.0"
```

Comment:
I dont know if we should avoid add this dependency only to use the NotOwner error. Can we replace it for a local Unauthorized {} enum variant on the error.rs?

Reply:
As these checks for the sender are the same in all contracts it is quite nice to reuse existing solutions for that. The mars-owner is just the new version of the Admin in cw-controllers. It has a better implementation and -- though not used here -- a quite safe mechanism for transfer of ownership.

Reply:
I was going to suggest to use cw-controllers but ok, sounds good!